render/helpers.go

```diff
@@ -95,3 +119,25 @@ func Markdown(md string) template.HTML {
 	s := blackfriday.Run([]byte(fmt.Sprintf("%s", modifiedMarkdown.String())))
 	return template.HTML(s)
 }
+
+// Localise localises text based on a key
+func Localise(key string, language string, plural int, templateArguments ...string) string {
+	if key == "" {
+		return ""
+	}
+	// Configure template data for arguments in strings
+	templateData := make(map[string]string)
+	for i, argument := range templateArguments {
+		stringIndex := strconv.Itoa(i)
+		key := "arg" + stringIndex
+		templateData[key] = argument
+	}
+	// Get translation
+	loc := i18n.NewLocalizer(bundle, language)
```

Comment:
Could we initialise each of the localisers in the init func and store them in a map?
```
map[string]localizer{
   "en": enLocalizer,
   "cy": cyLocalizer,
}
```
We could then reuse them instead of having to create a new one each time - bearing in mind this func will be called for each localized text on the page so it will be called a lot.

Reply:
Great idea! I have now created an InitLocalizer function within the helper. This should now only be created once and will use a map as suggested